numpy/__init__.pyi

```diff
@@ -36,6 +36,7 @@ from typing import (
     Container,
     Callable,
     Dict,
+    Final,
```

Comment:
Can you move the `Final` import to the lines below?

`Final` was only added to the `typing` module in python 3.8; 
for earlier versions it requires a backport from `typing_extensions`.

https://github.com/numpy/numpy/blob/29c58e632ce51e75e946014f076aba803ee57dde/numpy/__init__.pyi#L60-L63

Reply:
Done!